src/compas_timber/parts/beam.py

```diff
@@ -435,6 +416,10 @@ def from_endpoints(cls, point_start, point_end, width, height, z_vector=None, ge
         line = Line(point_start, point_end)
         return cls.from_centerline(line, width, height, z_vector, geometry_type)
 
+    @classmethod
+    def from_brep(cls, brep):
+        pass
```

Comment:
maybe `raise NotImplementedError` instead of passing?

Reply:
removed altogether. will be in it's own PR.